main/ServiceManager/ServiceManager.cpp

```diff
@@ -56,7 +58,10 @@ void MatterServiceRunner()
     MatterInterfaceHandler.SharedSemaphore = &MatterSemaphore;
     MatterInterfaceHandler.MatterAttributeUpdateCB = MatterAttributeUpdateCBMain;
     MatterInterfaceHandler.ConnectToMatterNetwork = MatterNetworkConnected;
-    Matter_TaskInit(&MatterInterfaceHandler);
+    Matter_TaskInit(&MatterInterfaceHandler, 
+                    &MatterHandle,
```

Comment:
Are you sure the `MatterHandle` should be defined in Main and then passed to the component?

Reply:
it was not defined in Main. also in ServiceManager.